schema/test/unit/tables/ciip_user_test.sql

```diff
@@ -69,16 +79,42 @@ select throws_like(
     'Administrator cannot delete rows from table_ciip_user'
 );
 
+select user_id from ggircs_portal.ciip_user_organisation where organisation_id=7;
+select * from ggircs_portal.ciip_user_organisation;
+
 -- CIIP_INDUSTRY_USER
+set jwt.claims.sub to '00000000-0000-0000-0000-000000000000';
 set role ciip_industry_user;
 select concat('current user is: ', (select current_user));
 
-select results_eq(
+select id from ggircs_portal.ciip_user;
+
+select isnt_empty(
   $$
-    select count(*) from ggircs_portal.ciip_user
+    select * from ggircs_portal.ciip_user where uuid=(select sub from ggircs_portal.session());
   $$,
-  ARRAY['7'::bigint],
-    'Industry user can view all data from ciip_user'
+    'Industry user can view their own data in the ciip_user table'
+);
+
+select isnt_empty(
+  $$
+    select * from ggircs_portal.ciip_user where id = 6;
+  $$,
+    'Industry user can view data from ciip_user table for users within the same approved organisation'
+);
```

Comment:
Not sure I'm totally following these `select *` tests, but they seem to check that users are able to see the things they should. I don't see they check that they can't see things they shouldn't (except for the `is_empty` one below).

Reply:
I've set up users 2 & 6 in a way that the current user (user 3) should be able to see that data, so isnt_empty asserts that something is returned when querying those user ids. I've set up user 1 in a way that user 3 should not be able to read that data, so when querying the table for user 1's data, it comes back empty (user 3 cannot see it).

Reply:
policies differ from grants where if a user queries for data that they're not allowed to see, no exception is thrown. The data just silently is not returned from the query.